### ISSUE TITLE
Magnus the Beast Claw at Writheblood Ruin 

### ISSUE THREAD

**Issue by JustAnOkapi** [RoundtableHold/roundtablehold.github.io#271]
title is missing from the bosses list
is an invasion that was added in 1.06

related to the second [White-Faced Varré](https://eldenring.wiki.fextralife.com/White+Mask+Varre) quest

**Comment by rmongiovi** [comment]
Yes.  It would be nice if this accommodated offline players who can't do the "three invasions" step in Varre's quest line.  You have to postpone that until you get to the Altus plateau and can substitute this invasion.
